src/pages/seulbiKim/Main/MainSb.js

```diff
@@ -1,7 +1,65 @@
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
+import Nav from '../../../components/Nav/Nav';
+import './MainSb.scss';
 
-function Main() {
-  return <div>Main</div>;
-}
+import FeedArticle from './component/FeedArticle';
+import MainRight from './component/MainRight';
 
-export default Main;
+const MainSb = () => {
+  const [comment, setComment] = useState('');
+  const [commentArr, setCommentArr] = useState([]);
+  const [feedList, setFeedList] = useState([]);
+  const nextId = useRef(4);
+  const handleSubmit = e => {
+    e.preventDefault();
+    if (e.target.comment.value !== '')
+      setCommentArr([
+        ...commentArr,
+        { id: nextId.current, name: 'seul', comment: comment },
+      ]);
+    e.target.comment.value = '';
+    nextId.current += 1;
+  };
+  console.log(commentArr);
+
+  const handleComment = e => {
+    e.preventDefault();
+    setComment(e.target.value);
+  };
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/seulbiCommentData.json')
```

Comment:
```suggestion
    fetch('data/seulbiCommentData.json')
```
로컬에 목데이터를 불러올 때는 `http://localhost:3000`은 입력하지 않아도 됩니다!